example/android/app/src/main/java/com/appcues/samples/reactnative/MainActivity.java

```diff
@@ -12,4 +14,10 @@ public class MainActivity extends ReactActivity {
   protected String getMainComponentName() {
     return "AppcuesReactNativeExample";
   }
+
+  // https://github.com/software-mansion/react-native-screens/issues/17#issuecomment-424704067
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(null);
+  }
```

Comment:
It is not totally clear to me why this is needed, but the Android side can randomly crash without this, as noted in this linked issue. Perhaps related to some change internally in the react native library dependencies. This has nothing at all to do with the core issue here and is just in our sample app. But it came up while I was testing out the fix, and I've seen once before in local testing.